_po4a-tools/po4a-create-all-targets.sh

```diff
@@ -103,10 +103,10 @@ process_with_po4a () {
         if [ $ext == yml ] ; then
             FILE_FORMAT=yaml
             OPTION="skip_array"
-        elif [ $ext == html ] ; then
+        elif [[ $ext == html || "$filename" == *'-index' ]] ; then
```

Comment:
Maybe add a comment here. also can you match on -index.md?

Reply:
Will add a comment.

Yes, it matches -index.md too because it only looks at the file name excluding the extension.